theme/static/css/hydroshare_core.css

```diff
@@ -4650,4 +4650,16 @@ div#resource-content {
 
 .btn-choose-coordinates {
 	vertical-align: top;
+}
+
+/* popover styling */
+.popover-content {
+  padding: 9px 14px;
```

Comment:
You don't need this padding here. These exact values are already specified by bootstrap.